deploys/demo-test.syncfor.science/config/defaults/clients.json

```diff
@@ -1,50 +0,0 @@
-[{
```

Comment:
It lives nowhere! Oops. I've created a fixture file in auth_proxy, https://github.com/sync-for-science/auth-proxy/pull/5. It creates a User and two Clients, research-app and test-suite. I don't think we need the other two anymore? I've moved it to the app itself, since all the different deploys were just using the same file anyways. It should still be easy to override if we need to though.